Yoast/wordpress-seo
# Add premium upsell modal for multiple keywords

## Summary

This PR can be summarized in the following changelog entry:

* Added a premium upsell modal for multiple keywords

## Relevant technical choices:

* Copied the method by which the modal is created from the implementation for the synonyms modal.

## Test instructions

This PR can be tested by following these steps:

* checkout this branch.
* Set up this branch:
   - `composer install`
   - `yarn install`
   - `Grunt build:js && grunt build:css`
   - `yarn start`
* Visit a post-edit page.
* Visit the metabox.
* Check below the focus keyword input field.
* Check that an additional link has been added with the text "Add additional keyword". Clicking this button should open a modal with the premium upsell. 
* Check if the content of the modal is valid, and if the modal is closable.
* Check if the link has been added to both the metabox and the sidebar.

## Quality assurance


Fixes #10517
